app/src/org/commcare/dalvik/activities/HomeButtons.java

```diff
@@ -136,6 +141,7 @@ public void onClick(View v) {
     private static View.OnClickListener getIncompleteButtonListener(final CommCareHomeActivity activity) {
         return new View.OnClickListener() {
             public void onClick(View v) {
+                reportButtonClick(GoogleAnalyticsFields.LABEL_INCOMPLETE_FORMS_BUTTON);
```

Comment:
Eh that's true in theory, but because of the hierarchical way in which the reporting works and how the product team will be making use of that, it makes sense to just provide it to them in both places.